src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -15,6 +15,23 @@ if (window.__osdStrictCsp__ && window.__osdCspNotEnforced__) {
   loadingMessage.style.display = 'flex';
 
   window.onload = function () {
+    /* ------------------------------- WAZUH ------------------------------- */
+    var interval = setInterval(() => {
+      var title = document.querySelector("#opensearch-dashboards-body > div > div.app-wrapper.hidden-chrome > div > div.application > div > ul > div.euiText.euiText--medium > div")
+      if (!!title) {
+        clearInterval(interval);
+        var content = document.querySelector("#opensearch-dashboards-body > div");
+        content.classList.add("wz-login")
+
+        // Don't force custom logo to have 100% width. It should be handled in the svg properties if needed
+        document
+          .querySelector('#opensearch-dashboards-body .wz-login figure.euiImage--fullWidth')
+          .classList.remove('euiImage--fullWidth')
+
+      }
+    })
+    /* ------------------------------- WAZUH ------------------------------- */
+
```

Comment:
**suggestion**: this could be removed in favor to apply the changes in the security plugin instead.

Reply:
``Resolved in these 2 PRs:
- https://github.com/wazuh/wazuh-dashboard/pull/50
 ```
var content = document.querySelector("#opensearch-dashboards-body > div");
        content.classList.add("wz-login")
```
https://github.com/wazuh/wazuh-dashboard/pull/50/files#diff-40e678cbfb32f3ef8ac92d63c508726015e95074c3bbf35efd79685b2de0e71aR69
- https://github.com/wazuh/wazuh-security-dashboards-plugin/pull/3
```
 // Don't force custom logo to have 100% width. It should be handled in the svg properties if needed
    document
       .querySelector('#opensearch-dashboards-body .wz-login figure.euiImage--fullWidth')
       .classList.remove('euiImage--fullWidth')
```
https://github.com/wazuh/wazuh-security-dashboards-plugin/pull/3/files#diff-b31eb00b43c58dd87689cbf4b98efbe85c489af9b4e85e1821161f72f18c639eR246